.github/workflows/release-nightly.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  schedule:
+    # https://crontab.guru/#0_12_*_*_1-5
+    # At 12:00 on every day-of-week from Monday through Friday.
+    - cron: '0 12 * * 1-5'
```

Comment:
This runs the script at noon, no?  Don't we want it to run sometime during the night? 

Or does this run in UTC time, and you accounted for the timezone difference?

Reply:
It will run at 8am EDT, 2pm CET currently. I stuck a time in there expecting to finalize it here.

> High-level question: how can we test this action works properly without releasing?

We cannot beyond the fact that I've at least confirmed the node script runs fine locally. I was thinking we should put in a time that is shortly after this PR merges, and we can make fixes if we hit an issue or if it works then we can change it to the hour we want it to release at.

Reply:
I think we should put the time that we want it to run, check whether it does work the following morning, and disable or make adjustments as needed if it doesn't work the first time

Reply:
We should also wait to merge it Monday, ideally

Reply:
Any preference for a time? I was thinking 8am would mean most of the team is waking up so we could address issues that same day (if any)

Reply:
Alternative suggestion, per slack: we can keep the time of expected merge and just make this a dry run, then do a follow-up PR to remove the dry run flag and set the time to the proper nightly time

Reply:
I've added the --dry-run flag to publish, so we can set a time to trigger, and see all the output to verify, then we can go back remove dry-run and set the final time we want to use

Reply:
I don't follow.  Is the idea to, prior to merging, set the cron to run at a time just in the future after we intend to merge it?

Reply:
Yep, when we're approved on everything else I can adjust the time to be what ever hour is closest after our merge, then we can pick up NODE-5336 to set the time and remove dry-run